api/service.go

```diff
@@ -71,16 +69,17 @@ func registerServices(ctx context.Context, grpcServer *grpc.Server, gwmux *runti
 	}
 
 	// register other api endpoints
-	err = registerAPIs(ctx, invoiceUnpaidService, configService, nodeObjReg, grpcServer, gwmux, addr, dopts)
+	err = registerAPIs(ctx, invoiceUnpaidService, configService, grpcServer, gwmux, addr, dopts)
 	if err != nil {
 		return nil, err
 	}
 
 	cfg := nodeObjReg[bootstrap.BootstrappedConfig].(config.Configuration)
-	return httpapi.Router(cfg, configService, invoiceUnpaidService.(documents.TokenRegistry), docService), nil
+	jobsMan := nodeObjReg[jobs.BootstrappedService].(jobs.Manager)
+	return httpapi.Router(cfg, configService, invoiceUnpaidService.(documents.TokenRegistry), docService, jobsMan), nil
```

Comment:
Maybe we can think about making this a routerConfig argument, so the function call doesnt get too long.

Reply:
I was thinking about waiting how it will evolve and make the change suitable at that point